org/views.py

```diff
@@ -54,6 +54,50 @@ def post(self,request):
             data = serializer.errors
             return Response(data, status.HTTP_400_BAD_REQUEST)
 
+class GroupView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    @swagger_auto_schema(
+        operation_id='create_group',
+        request_body=CreateGroupSerializer,
+        responses={
+            '201': set_example({}),
+            '400': set_example(responses.create_group_400),
+            '401': set_example(responses.create_group_401),
+            '403': set_example(responses.create_group_403),
+            '404': set_example(responses.create_group_404)
+        },
+    )
+    def post(self,request, org_id):
+        """
+        When a user posts the required details to this API, a number of checks follow:
+        1. A check at the URL, that the org_id in the url exists or not(404).
+        2. Whether the authorized user is a member of the organisation(401).
+        3. Whether the authorzed member is an admin of the organisation(403).
+        4. Whether the data posted is valid or not(400).
+        5. If everything goes right, we create a unique route slug in the overidden save()
+            which takes org_id as a parameter(201).
+        """
+        try:
+            org = Org.objects.get(id=org_id)
+        except:
```

Comment:
Catch the specific exception, here the specific exception is `Org.DoesNotExist`.
Same as pointed in sahil's PR [here](https://github.com/ECellNitrr/teesco-backend/pull/96#discussion_r432829048)